adapter/memQuota/memQuota_test.go

```diff
@@ -25,28 +24,7 @@ import (
 	at "istio.io/mixer/pkg/adapter/testing"
```

Comment:
suggestion: we rename the `//pkg/adapter/testing` package to avoid the collision with the standard `testing` package. Since they will only ever be used together, it would be nice to not have to rename it on every import. 

How about `//pkg/adapter/test` ?  That would give us code that read like `test.NewEnv(t)`.